docs/contexts/traceability-v1.jsonld

```diff
@@ -122,6 +122,173 @@
         }
       }
     },
+    "EcommerceInvoiceRegistrationCredential": {
+      "@id": "https://w3id.org/traceability/EcommerceInvoiceRegistrationCredential",
```

Comment:
This link is wrong... it should be https://w3id.org/traceability#EcommerceInvoiceRegistrationCredential

the IRI should lead to a human readable defintion... this problem is everywhere in the repo, we need to stop merging PRs that contribute broken links.

Reply:
fully agree, all links were updated